apiConfig.js

```diff
@@ -0,0 +1,97 @@
+const BASE_API_URL = "https://fakerestapi.azurewebsites.net/api/v1";
+
+const endpoints = {
+  activities: {
+    getAll: {
+      url: `${BASE_API_URL}/Activities`,
```

Comment:
No need to rename this object if we removed the `method` property. We can set the name back to `urls` and remove the inner `url` properties. In the end, you should get the urls by calling something like `urls.activites.getAll`

Reply:
done